src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/Målgruppe.tsx

```diff
@@ -58,45 +58,45 @@ const Målgruppe: React.FC<{ grunnlag: VilkårperioderGrunnlag | undefined }> =
             rundskrivlenke={rundskrivMålgruppe}
         >
             <FlexColumn gap={2}>
-                <RegisterYtelser grunnlag={grunnlag} />
+                <RegisterYtelser
+                    grunnlag={grunnlag}
+                    lagRadForPeriode={(valgtPeriode: YtelseGrunnlagPeriode) =>
+                        settRadIRedigeringsmodus(undefined, valgtPeriode)
```

Comment:
Ikke så fan av at jeg må sende inn undefined her, men kjipt å skulle ha to ulike funksjoner for det samme :/